backend/app/services/readers/users.py

```
@@ -0,0 +1,142 @@
+    base = UserReader()
+
+    if settings.SERVICE_EXTENSION:
+        try:
+            import importlib
+
+            ext = importlib.import_module(f"{settings.SERVICE_EXTENSION}.users")
+            result = ext.wrap(base)
+            if result:
+                logger.info("User reader extension loaded")
+                return result
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)